flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/scalingpolicy/EnforceParallelismChangeRescalingController.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.scheduler.adaptive.scalingpolicy;
+
+import org.apache.flink.runtime.jobgraph.JobVertexID;
+import org.apache.flink.runtime.scheduler.adaptive.allocator.VertexParallelism;
+
+/**
+ * Simple scaling policy for a reactive mode. It just checks that the new parallelism is different
```

Comment:
Same question about this `reactive mode`.

https://github.com/apache/flink/pull/22985#discussion_r1320535556

Reply:
please see above comment

Reply:
Sorry, I didn't see any new comment related to `reactive mode`, please correct my if I missed any comment, thanks~

https://github.com/apache/flink/pull/22985#discussion_r1320535556

Reply:
in that case, we need to remove `for a reactive mode` in both the scaling policies (`EnforceMinimalIncreaseRescalingController` and `EnforceParallelismChangeRescalingController`)